chain-signatures/node/src/protocol/cryptography.rs

```diff
@@ -407,20 +407,23 @@ impl CryptographicProtocol for RunningState {
             tracing::warn!(?err, "running: failed to stockpile presignatures");
         }
         drop(triple_manager);
-        for (p, msg) in presignature_manager.poke() {
+        for (p, msg) in presignature_manager.poke().await {
             let info = self.fetch_participant(&p)?;
             messages.push(info.clone(), MpcMessage::Presignature(msg));
         }
 
         crate::metrics::NUM_PRESIGNATURES_MINE
             .with_label_values(&[my_account_id.as_str()])
-            .set(presignature_manager.my_len() as i64);
+            .set(presignature_manager.count_mine().await as i64);
         crate::metrics::NUM_PRESIGNATURES_TOTAL
             .with_label_values(&[my_account_id.as_str()])
-            .set(presignature_manager.len() as i64);
+            .set(presignature_manager.count_all().await as i64);
         crate::metrics::NUM_PRESIGNATURE_GENERATORS_TOTAL
             .with_label_values(&[my_account_id.as_str()])
-            .set(presignature_manager.potential_len() as i64 - presignature_manager.len() as i64);
+            .set(
+                presignature_manager.count_potential().await as i64
+                    - presignature_manager.count_all().await as i64,
+            );
```

Comment:
ok, this naming is now pretty confusing. When I see `count_all` I would assume it includes potential as well. This calculation tells me it does not, but then `all` is not inclusive and is confusing.

Not a fan of the word `count` word because `len` is so much more standard in rust. I now don't see a reason to change what we had before.

I say we have this consistent naming instead:
- `len`: number of triple/presignatures
- `len_mine`: number of mine triple/presignatures
- `len_generators`: number of generators
- `len_potential`: current number of triple/presignature + generators

Reply:
I agree that `len()` is a good name when we work directly with a map. But here it is usually called as
```
triple_manager.count_mine()
```
In this conext `len()` is a bit ambiguous.

Agree about `count_all()`, will rename it to `count()` or maybe `count_generated()` to differentiate between generation protocols and pre-signatures that are ready to use.

Reply:
Hm, if we add `_generated` it make sense with `len` too.